utils/viewmodel/src/main/kotlin/com/bumble/appyx/utils/viewmodel/node/ViewModelNode.kt

```diff
@@ -0,0 +1,33 @@
+package com.bumble.appyx.utils.viewmodel.node
+
+import androidx.lifecycle.ViewModelStore
+import androidx.lifecycle.ViewModelStoreOwner
+import com.bumble.appyx.navigation.lifecycle.DefaultPlatformLifecycleObserver
+import com.bumble.appyx.navigation.modality.BuildContext
+import com.bumble.appyx.navigation.node.Node
+import com.bumble.appyx.utils.viewmodel.ActivityIntegrationPointWithViewModel
+
+class ViewModelNode(
```

Comment:
Otherwise you can't extend
```suggestion
open class ViewModelNode(
```